openslides_backend/services/datastore/adapter.py

```diff
@@ -263,6 +268,10 @@ def min(
             f"Start MIN request to datastore with the following data: {command.data}"
         )
         response = self.retrieve(command)
+        if lock_result:
+            self.update_locked_fields(
+                CollectionField(collection, field), response.get("position")
+            )
         return response
```

Comment:
We should probably adjust the adapter (for this and the other aggregate functions) to just return the min and not the position. If you want, you can do this here or open another PR for it.

Reply:
Done